sdk/keyvault/keyvault-keys/src/core/models/parameters.ts

```diff
@@ -20,6 +20,27 @@ export const acceptLanguage: coreHttp.OperationParameter = {
     }
   }
 };
+
+export const spanOptions: coreHttp.OperationParameter = {
+  parameterPath: ["options", "requestOptions", "spanOptions"],
+  mapper: {
+    serializedName: "span-options",
+    type: {
+      name: "Object"
+    }
+  }
+};
+
+export const spanOptionsV2: coreHttp.OperationParameter = {
+  parameterPath: ["options", "spanOptions"],
+  mapper: {
+    serializedName: "span-options",
+    type: {
+      name: "Object"
+    }
+  }
```

Comment:
Do we need the v2 anymore?

Reply:
Let me give you an example in ```getKey``` method, when you are calling ```this.client.getKey```, you are passing the ```options``` value as parameter. But in ```getDeletedKey``` method, when you are calling ```this.client.getDeletedKey```, you are passing ```options.requestOptions```. In order to handle this difference we have spanOptions & spanOptionsV2. Essentially they are the same. Only the path value where to find the spanOptions is different.

Reply:
I believe that is a bug with the `getKey` method. It should be passing request options as well.
I have created a PR to fix this issue https://github.com/Azure/azure-sdk-for-js/pull/4656
and updated the keyvault tracing PR #4654 to not have these 2 different versions of "span options" in parameters.ts file